primitives/src/lib.rs

```diff
@@ -7,6 +7,7 @@ mod amt;
 mod hamt;
 mod link;
 
+pub use address::*;
```

Comment:
Can we call the file `taddress.rs` and the mod `taddress`? So we differentiate it from `fvm::shared::address`. Apart from that LGTM.

Reply:
changed!